code/modules/research/designs/medical_designs.dm

```diff
@@ -425,6 +425,18 @@
 	build_path = /obj/item/organ/internal/cyberimp/arm/toolset_abductor
 	category = list("Medical")
 
+/datum/design/cyberimp_surgical_abductor
+	name = "Abductor Surgical Toolset Implant"
+	desc = "An alien surgical toolset, designed to be installed on the subject's arm."
+	id = "ci-med-abductor"
+	req_tech = list("materials" = 6, "engineering" = 6, "plasmatech" = 6, "abductor" = 3)
```

Comment:
I dont know much about tech levels but i think it should probably have "biotech" and "magnet" instead of "plasmatech"? The ones over the surgical toolset instead of engineering one.

Reply:
Whoops, forgot to push it when I was working on this.